modules/ROOT/pages/_partials/symbolicate-ios.adoc

```diff
@@ -0,0 +1,79 @@
+== Symbolication
+
+In order to make sense of a crash report you will need to symbolicate
+ its content.
+Symbolication being the process of replacing memory addresses in a crash report with human-readable function names and line numbers.
+To do this you need to have debug symbol (`.dSYM`) files for the application and for Couchbase Lite's framework.
+
+== Finding local dSYM
+
+To locate the local dSYM files:
+
+* For scheme based builds, they are located in the derived data path
+* For the target based builds, they are located in the configuration build directory path, as specified in the _target_ setting
+* For archives, they are located in `.xcarchive`.
+
+TIP: If you still cannot locate the file, then use `mdfind` as described in the
+_Symbolication Troubleshooting_ section of the _Apple Technical Note TN2151 --
+Understanding and Analyzing Application Crash Reports_: https://developer.apple.com/library/archive/technotes/tn2151/_index.html
+
+== The CouchbaseLite dSYM
+
+Couchbase Lite's `dSYM` folder is located in the same directory as the Couchbase Lite binary framework (this also applies for _Cocoapod_ and _Carthage_).
+It has the same filename as framework but has a `.dSYM` extension.
+
+.dSYM in CouchbaseLiteSwift
+====
+----
+couchbase-lite-swift_enterprise_2.7.0/iOS/CouchbaseLiteSwift.framework.dSYM
+----
+====
+
+== Bitcode enabled Apps
+
+Apple creates new dSYM files for apps uploaded to the _App Store_ or _TestFlight_ with _Bitcode_ enabled.
```

Comment:
I understand that Apple doesn't include bitcode to the app anymore, but how the developers can get the dSYM file from the archived app or from the App Store is still valid. So I think this section can be rewritten such as "How to get the dSYM file from the app."